back-end/src/main/java/com/example/demo/entity/Vacancy.java

```diff
@@ -0,0 +1,39 @@
+package com.example.demo.entity;
+
+import com.example.demo.entity.common.JPAEntity;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.data.annotation.CreatedDate;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Data
+@Entity
+@EqualsAndHashCode(of = "id", callSuper = false)
+@ToString(of = "id", callSuper = false)
+public class Vacancy extends JPAEntity {
+
+    @Column(name = "vacancies")
+    private Integer vacancies;
+
+    @Column(name = "post_by", updatable = false)
+    @CreatedBy
+    private String postBy;
```

Comment:
you use Created By this mean Hold user name login user ? if yes you need to create users table also to make sue insert data will valid

Reply:
I mean by postBy the employee that made this vacancy